src/v/redpanda/admin/api-doc/debug_bundle.json

```diff
@@ -4,15 +4,62 @@
     "summary": "Start an rpk debug bundle",
     "operationId": "post_debug_bundle",
     "consumes": [
-        "application/json"
+      "application/json"
     ],
     "parameters": [
       {
         "name": "config",
         "in": "body",
         "required": true,
         "schema": {
-          "$ref": "#/definitions/bundle_start_config"
+          "description": "Parameters to kick off debug bundle generation",
+          "type": "object",
+          "required": [
+            "job_id"
+          ],
+          "properties": {
+            "job_id": {
+              "description": "The Job UUID to use",
+              "type": "string"
+            },
+            "config": {
+              "description": "The parameters",
+              "type": "object",
+              "required": [],
+              "properties": {
+                "authentication": {
+                  "description": "Authentication object",
+                  "type": "scram_authn"
+                },
+                "controller_logs_size_limit_bytes": {
+                  "type": "int"
+                },
```

Comment:
I'd separate the bulk of the movement into its own commit, and I'd also inline `scram_authn` here, too.

And since we're not codegen, we can use the correct swagger types, `int` -> `integer`

This is an example of the final result: https://github.com/redpanda-data/redpanda/commit/214e6f577e80581cc9d0fbf601de2bddd3a79226